client/cypress/integration/user-list.spec.ts

```diff
@@ -0,0 +1,147 @@
+import { UserListPage } from '../support/user-list.po';
+
+const page = new UserListPage();
+
+describe('User list', () => {
+
+  before(() => {
+    cy.task('seed:database');
+  });
+
+  beforeEach(() => {
+    page.navigateTo();
+  });
+
+  it('Should show 10 users in both card and list view', () => {
+    page.getUserCards().should('have.length', 10);
+    page.changeView('list');
+    page.getUserListItems().should('have.length', 10);
+  });
+
+  it('Should type something in the name filter and check that it returned correct elements', () => {
+    // Filter for user 'Lynn Ferguson'
+    cy.get('#user-name-input').type('Lynn Ferguson');
+
+    // All of the user cards should have the name we are filtering by
+    page.getUserCards().each(e => {
+      cy.wrap(e).find('.user-card-name').should('have.text', 'Lynn Ferguson');
+    });
```

Comment:
What do you think about some dollar signs here (just for consistency)?

```suggestion
    page.getUserCards().each($el => {
      cy.wrap($el).find('.user-card-name').should('have.text', 'Lynn Ferguson');
    });
```

Reply:
I'm personally in favor of using the dollar sign when it's referring to a JQuery object but @NicMcPhee didn't seem to like doing that so maybe we actually remove it from the ones that currently have it.

Reply:
Sounds good, that works too! :)